merkle/hstar2_test.go

```diff
@@ -35,22 +34,15 @@ const treeID = int64(0)
 
 var (
 	deB64 = testonly.MustDecodeBase64
+	hd    = testonly.MustHexDecode
 )
 
-// Some known answers for incrementally adding index/value pairs to a sparse tree.
-// rootB64 is the incremental root after adding the corresponding i/v pair, and
-// all i/v pairs which come before it.
-//
-// Values were calculated with the C++/Python sparse Merkle tree code in the
-// github.com/google/certificate-transparency repo.
-// TODO(alcutter): replace with hash-dependent computation. How is this computed?
 var simpleTestVector = []struct {
 	index, value, root []byte
 }{
-	{nil, nil, deB64("xmifEIEqCYCXbZUz2Dh1KCFmFZVn7DUVVxbBQTr1PWo=")}, // Empty tree.
```

Comment:
Is there a reason to delete the empty tree test? I guess, it should also have some "empty tree" hash value.